pontoon/static/js/widgets/checkbox.js

```diff
@@ -0,0 +1,38 @@
+
+import React from 'react';
+
+
+export default class Checkbox extends React.Component {
+
+    /* A checkbox which you can set `indeterminate` on
+     *
+     */
+
+    constructor (props) {
+        super(props);
+        this.el = {};
+    }
+
+    get indeterminate () {
+        const {indeterminate} = this.props
+        if (indeterminate) {
+            return true;
+        }
```

Comment:
Why don't you just do `return this.props.indeterminate;`? If the goal is to not expose the prop, then maybe `return this.props.indeterminate ? true : false;`?

Reply:
i think i wanted to leave it undefined if its not set in the props

Reply:
which i think is what it would be without messing with it, i can check again

Reply:
actually - i checked its `false` by default - ill update...